cmd/util/cmd/execution-state-extract/cmd.go

```diff
@@ -346,6 +342,12 @@ func run(*cobra.Command, []string) {
 			hex.EncodeToString(stateCommitment[:]),
 			flagExecutionStateDir,
 		)
+
+		err := ensureCheckpointFileExist(flagExecutionStateDir)
+		if err != nil {
+			log.Error().Err(err).Msgf("cannot ensure checkpoint file exist in folder %v", flagExecutionStateDir)
```

Comment:
Some test cases wants the checkpoint folder to be empty. Changed it from Fatal to Error for now.